src/app/home-page/home-logged-in/home-logged-in.component.html

```diff
@@ -14,10 +14,10 @@
   ~    limitations under the License.
   -->
 
-<div fxLayout>
+<div fxLayout *ngIf="!Dockstore.FEATURES.enableNewDashboard; else newDashboard">
   <app-my-sidebar fxFlexAlign="stretch"></app-my-sidebar>
 
-  <div id="column-holder" fxLayout="row" fxLayout.lt-md="column" fxLayoutAlign="start start" fxLayoutGap="10px">
+  <div id="column-holder" fxLayout="row" fxLayout.lt-md="column" fxLayoutAlign="start start" fxLayoutGap="10px" fxFlex="1 2 100%">
```

Comment:
You removed all your other changes to the old dashboard, but this still remains. Just making sure it's intentional? Maybe it's required because of the new div this is child of?

Reply:
Yeah without it, it caused everything to be shifted towards the left and did not fill the screen, I thought that it would behave as before if I just added the `*ngIf` into the original div instead of making a new parent div, but the problem still persisted so I had to add the new `flex` definitions.